logicalplan/plan.go

```diff
@@ -185,6 +185,23 @@ func replaceSelectors(plan parser.Expr) parser.Expr {
 			*current = &MatrixSelector{MatrixSelector: t, OriginalString: t.String()}
 		case *parser.VectorSelector:
 			*current = &VectorSelector{VectorSelector: t}
+		case *parser.Call:
```

Comment:
Will this code be executed if the MergeSelectors optimizer is disabled?

Reply:
It runs before optimizers I think

Reply:
Got it, I got it confused with something else.